### PR TITLE
Network speed monitor

### PR DESCRIPTION
**Pull request by muriloventuroso**
Add support to network speed monitor, with display control.

### CODE REVIEW

File: src/Widgets/MainView.vala [hunk 1]
```diff
@@ -31,8 +31,8 @@ public class SysMonitor.Widgets.MainView : Gtk.Grid {
         attach (ram_value_label,    2, 1, 1, 1);
         attach (swap_label,         1, 2, 1, 1);
         attach (swap_value_label,   2, 2, 1, 1);
-        attach (uptime_label,       1, 3, 1, 1);
-        attach (uptime_value_label, 2, 3, 1, 1);
```
Comment (PlugaruT) [inline src/Widgets/MainView.vala]: I am not sure why did you remove the uptime information? Just add the network info above the uptime row.

File: src/Widgets/DisplayWidget.vala [hunk 2]
```diff
@@ -16,6 +21,10 @@ public class SysMonitor.Widgets.DisplayWidget : Gtk.Grid {
         icon = new Gtk.Image.from_icon_name ("computer-symbolic", Gtk.IconSize.MENU);
         cpu_label = new Gtk.Label ("CPU");
         ram_label = new Gtk.Label ("MEM");
+        network_down_label = new Gtk.Label ("DOWN");
```
Comment (PlugaruT) [inline src/Widgets/DisplayWidget.vala]: these two labels are nice, but because the network info is changing very fast and the values have from 0 to 4 digits, it doesn't look nice and is kind of visually disturbing when all the indicators move when network info changes. Maybe you can find a way to set an optimal fixed width of the label so that the possible values could fit in there. This is also referring to the PopoverWidget.

File: src/Widgets/MainView.vala (hunk 1 above)
Comment (muriloventuroso) [reply src/Widgets/MainView.vala]: Corrected, I also do not know why I did this.

File: src/Widgets/DisplayWidget.vala (hunk 2 above)
Comment (muriloventuroso) [reply src/Widgets/DisplayWidget.vala]: The size is now fixed, the indicator does not move anymore. It's a lot better.
